mod-invoice-storage/schemas/voucher.json

```diff
@@ -24,7 +50,19 @@
       "description": "Export this payment record to a financial system",
       "type": "boolean"
     },
-    "voucherType":{
+    "status": {
+      "description": "Status of the voucher",
+      "type": "string",
+      "enum": [
+        "Awaiting Payment",
```

Comment:
We need to start using sentence case... "Awaiting payment"

Reply:
Corrected. Using sentence case for voucher `type` and `status`:
`Awaiting Payment` -> `Awaiting payment`
`Pre-Payment` -> `Pre-payment`